<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoMng</class>
 <widget class="QDialog" name="UserInfoMng">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>数据查询</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,2,2,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>用户ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_id"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>用户名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>所属部门</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cm_dept"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnQuery">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder></zorder>
     <zorder>btnQuery</zorder>
     <zorder></zorder>
     <zorder></zorder>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tvIns">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
